documentation/docs/release-notes/3.2.0.md

```diff
@@ -84,6 +84,8 @@ Initially, Nomad will assist with managing monitoring agents, enhancing service
 
 - [PMM-13824](https://perconadev.atlassian.net/browse/PMM-13824) - Updated **MongoDB Replicaset summary dashboard** and **MongoDB Sharded cluster summary** dashboard with a new panel to show current Feature Compatibility Version. We've also added panels to show Nodes, Databases count and Last election time.
 
+- [PMM-13210](https://perconadev.atlassian.net/browse/PMM-13210) - Upgraded VictoriaMetrics to version 1.114.0, which includes several performance improvements and bug fixes. This upgrade enhances the overall performance and reliability of PMM's metrics storage and retrieval.
```

Comment:
```suggestion
- [PMM-13210](https://perconadev.atlassian.net/browse/PMM-13210) - Upgraded VictoriaMetrics to version 1.114.0. This latest version includes key upstream improvements and fixes, ensuring more reliable metric collection, storage, and faster data access in PMM.
```

Reply:
it's already not the latest :) They've released 1.116.0 recently.

Reply:
ok, I've updated the suggestion and approved